<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QArkOptimizationParameterWidget</class>
 <widget class="QWidget" name="QArkOptimizationParameterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>39</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLineEdit" name="valueLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="optimizeCheckBox">
     <property name="text">
      <string>Optimize</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="minCheckBox">
     <property name="text">
      <string>Min</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="minLineEdit"/>
   </item>
   <item>
    <widget class="QCheckBox" name="maxCheckBox">
     <property name="text">
      <string>Max</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="maxLineEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
